themes/default/content/_index.md

```diff
@@ -40,39 +40,193 @@ overview:
+                if err != nil {
+                  return err
+                }
+
+                // Export some values in case they are needed elsewhere
+                ctx.Export("kubeconfig", eksCluster.Kubeconfig)
+                ctx.Export("vpcId", eksVpc.VpcId)
+                return nil
+              })
+            }
+    - title: MyStack.cs
+      language: csharp
+      code: |
+        using Pulumi;
+        using Awsx = Pulumi.Awsx;
+        using Eks = Pulumi.Eks;
+        using System.Collections.Generic;
+
+        return await Deployment.RunAsync(() =>
+        {
+            // Create a new VPC
+            var eksVpc = new Awsx.Ec2.Vpc("eks-vpc", new()
+            {
+                EnableDnsHostnames = true,
+                CidrBlock = "10.0.0.0/16",
+            });
+
+            // Create the EKS cluster
+            var eksCluster = new Eks.Cluster("eks-cluster", new()
+            {
+                VpcId = eksVpc.VpcId,
+                PublicSubnetIds = eksVpc.PublicSubnetIds,
+                PrivateSubnetIds = eksVpc.PrivateSubnetIds,
+                InstanceType = "t2.medium",
+                DesiredCapacity = 3,
+                MinSize = 3,
+                MaxSize = 6,
+                NodeAssociatePublicIpAddress = false,
+            });
+
+            // Export some values for use elsewhere
+            return new Dictionary<string, object?>
+            {
+                ["kubeconfig"] = eksCluster.Kubeconfig,
+                ["vpcId"] = eksVpc.VpcId,
+            };
+        });
+    - title: Pulumi.yaml
+      language: yaml
+      code: |
+        name: pulumi-eks
+        description: A simple EKS cluster
+        runtime: yaml
+        resources:
+          # Create a VPC for the EKS cluster
+          eks-vpc:
+            type: awsx:ec2:Vpc
+            properties:
+              enableDnsHostnames: true
+              cidrBlock: "10.0.0.0/16"
+          # Create the EKS cluster
+          eks-cluster:
+            type: eks:Cluster
+            properties:
+              vpcId: ${eks-vpc.vpcId}
+              publicSubnetIds: ${eks-vpc.publicSubnetIds}
+              privateSubnetIds: ${eks-vpc.privateSubnetIds}
+              instanceType: "t2.medium"
+              desiredCapacity: 3
+              minSize: 3
+              maxSize: 6
+              nodeAssociatePublicIpAddress: false
+        outputs:
+          # Output the Kubeconfig for the cluster
+          kubeconfig: ${eks-cluster.kubeconfig}
+          vpcId: ${eks-vpc.vpcId}
 
 deploy:
-  title: Deploy
+  title: Get to production faster
   description: |
-    Deploy cloud applications and infrastructure faster and with confidence, using one shared approach that works for day one and beyond for the entire team.
+    Pulumi gives you a faster outer dev loop by making  CI/CD for your IaC seamless and the default experience. Pulumi has integrations with all the popular CI/CD platforms and testing frameworks.
+
   items:
```

Comment:
I think we should delete out all these sub-bullets to lighten up on text. 

Add in the Get Started button